src/apiCalls.js

```diff
@@ -0,0 +1,97 @@
+export const getAllMovies = () => {
+  return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+      .then ( response => {
+      if (response.ok) {
+        return response.json()
+      } else {
+        throw Error
+      }
+    })
+    .then(data => {
+      data.movies.sort((a,b)=>{
+       return b.average_rating - a.average_rating
+      })
+      return({loading: '', movies: data.movies, error: '' })
+    })
+    .catch(error => {
+      console.log(error)
+      return({ error: 'We encountered an error, please reload page' });
+    }
+    );
+}
```

Comment:
Great idea to keep all the logic inside the apiCalls files and then just resolving it inside of the main file. Keeps a lot of logic out of the components which keeps them looking clean.